src/styles/blocks/card__code.scss

```diff
@@ -0,0 +1,15 @@
+.card__code {
+  margin-bottom: 16px;
+  color: #616070;
+  width: 98px;
+  height: 14px;
+  top: 247px;
+  left: 17px;
```

Comment:
The `top` and `left` properties will have no effect unless the element has a positioning context. Consider adding `position: relative;` or another positioning property if needed.